<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>375</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>675</width>
    <height>375</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1234566</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Импорт из DXF (СПБ)</string>
  </property>
  <property name="accessibleName">
   <string notr="true">btnSave</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="buttonClose">
         <property name="text">
          <string>Закрыть</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="1" colspan="4">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="2">
            <widget class="QPushButton" name="buttonAddSourceFile">
             <property name="maximumSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Журнал операции:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="buttonRemoveSourceFile">
             <property name="maximumSize">
              <size>
               <width>23</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="3" colspan="2">
            <widget class="QListWidget" name="listView"/>
           </item>
          </layout>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="buttonImport">
           <property name="text">
            <string>Импортировать</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QgsFileWidget" name="fileWidgetObj">
           <property name="accessibleDescription">
            <string/>
           </property>
           <property name="fileWidgetButtonVisible">
            <bool>true</bool>
           </property>
           <property name="dialogTitle">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Выберите DXF-файл</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>buttonImport</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>buttonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
